.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  push:
+    tags:
+    - 'v*'
+  workflow_dispatch:
+    
+name: Upload Release Assets
+
+jobs:
+  build:
+    name: Upload Release Assets
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
```

Comment:
Not sure if it's possible with Github Action, but it would be better `prerelease` is set to `true` when the tag contains `rc`.